SD-K9/src/app/pages/app-settings/app-settings.page.html

```diff
@@ -12,16 +12,10 @@
 <ion-content>
   <ion-list>
   <ion-item>
-    <ion-label>Personal Settings</ion-label>
+    <ion-button (click)="presentAlert()">Privacy Policy</ion-button>
   </ion-item>
   <ion-item>
-    <ion-label>Notifications</ion-label>
-  </ion-item>
-  <ion-item>
-    <ion-label>Privacy Policy</ion-label>
-  </ion-item>
-  <ion-item>
-    <ion-label>About</ion-label>
+    <ion-button (click)="changeTheme()">ColorBlind Theme</ion-button>
```

Comment:
Would it be possible to have a toggle instead to see when it is selected ?

Reply:
Thanks for the feedback! Implemented!